graphium/data/sampler.py

```diff
@@ -0,0 +1,43 @@
+import torch.utils.data as data_utils
+import numpy as np
+import time
+import os
+import torch
+
+
+class CustomSampler(data_utils.Sampler):
+    def __init__(self, dataset, sampler_task_dict, data_path, data_hash):
```

Comment:
Please add the docstring here to explain what the class does.

Also, define the type of each of the inputs. Something like that
```dataset: Dataset, sampler_task_dict: Dict[str, Optional[float]], data_path: PathLike, data_hash: str```